sql/patches/patch-revoke-permissions.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE /*$wgDBprefix*/mw_permissions
+  ADD COLUMN perm_revoke LONGTEXT AFTER perm_autopromote;
```

Comment:
`AFTER` does not match with `mw_permissions.sql`

Reply:
How so?

Reply:
On `mw_permissions.sql` `perm_revoke` was added after `perm_permissions` but here it's added after `perm_autopromote`

Reply:
Oh, crap your right. Thanks for pointing that out!